site-scrapers/Harrington-acuity/config.js

```diff
@@ -0,0 +1,43 @@
+const site = {
+    name: "Harrington (local residents only)",
+    street: "153 Chestnut Street",
+    city: "Southbridge",
+    zip: "01550",
+    website:
+        "https://app.acuityscheduling.com/schedule.php?owner=22192301&calendarID=5202050",
+    restrictions:
+        "Registration is open to only those who live in the following towns: Auburn, Brimfield, Brookfield, (North, East, West) Charlton, Dudley, Holland, Leicester, Oxford, Southbridge, Spencer, Sturbridge, Sutton, Uxbridge, Wales, Warren, Webster",
+};
+
+const locations = [
+    {
+        name: "Harrington &emdash; local residents",
+        street: "153 Chestnut Street",
+        city: "Southbridge",
+        zip: "01550",
+        website:
+            "https://app.acuityscheduling.com/schedule.php?owner=22192301&calendarID=5202050",
+        restrictions:
+            "Registration is open to only those who live in the following towns: Auburn, Brimfield, Brookfield, (North, East, West) Charlton, Dudley, Holland, Leicester, Oxford, Southbridge, Spencer, Sturbridge, Sutton, Uxbridge, Wales, Warren, Webster",
+        signUpLink:
+            "https://app.acuityscheduling.com/schedule.php?owner=22192301&calendarID=5202050",
+    },
+];
+
+const site2 = {
```

Comment:
Is there a reason you're not using this yet?

Reply:
Yes. Initially, I started with trying to do both at the same time, but ran into some weird problem. So, it was a divide and conquer strategy, picking the local residents one to get done first before iterating through both sites. Local residents has been showing availability, whereas the non-local has not. So, I thought it would be better to get the one that's likely to have availability up first and then iterate. I moved the non-local off to `const site2` to save the config into for later.